api/api.go

```diff
@@ -36,8 +36,8 @@ var (
 
 // API is our API service
 type API struct {
-	rtfs.Manager
-	rtfs.KeystoreManager
+	ipfs    rtfs.Manager
+	keys    rtfs.KeystoreManager
```

Comment:
this might need to be a pointer - `rtfs.Manager` is an interface so is already a pointer, whereas `KeystoreManager` is a concrete type